src/server/scripts/Commands/cs_npc.cpp

```diff
@@ -162,7 +162,8 @@ EnumName<CreatureFlagsExtra> const flagsExtra[FLAGS_EXTRA_COUNT] =
     CREATE_NAMED_ENUM(CREATURE_FLAG_EXTRA_NO_SKILLGAIN),
     CREATE_NAMED_ENUM(CREATURE_FLAG_EXTRA_TAUNT_DIMINISH),
     CREATE_NAMED_ENUM(CREATURE_FLAG_EXTRA_ALL_DIMINISH),
-    CREATE_NAMED_ENUM(CREATURE_FLAG_EXTRA_DUNGEON_BOSS)
+    CREATE_NAMED_ENUM(CREATURE_FLAG_EXTRA_DUNGEON_BOSS),
+    CREATE_NAMED_ENUM(CREATURE_FLAG_EXTRA_NO_PULL_BY_GRIP)
```

Comment:
It doens't strictly have to but it will cause this flag to be printed last in chat if there are more flags set.